src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/Contracts/RenameRequest.cs

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement.Contracts
+{
+    /// <summary>
+    /// Property class for ObjectManagement Service
```

Comment:
Remove - it's incorrect and generally not useful enough to bother adding doc comments for (name is clear enough)